CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # master
 
+# 0.3.4
+
+* Add source_id to `Account` and assign it when app is being installed. In order to make this work:
+
+1. Update your `Gemfile`
```

Comment:
Isint this a little too verbose to be in the changelog?
Maybe just paste it in PR comment, so it can be referenced in app PRs?